docs/modules/getting-started/pages/security.adoc

```diff
@@ -0,0 +1,341 @@
+. Run `go mod init <name of the directory>`.
+. Run `go get github.com/hazelcast/hazelcast-go-client`.
+. While in this directory, create a `go` file (for example `main.go`) containing the following code.
++
+[source,go]
+----
+package main
+
+import (
+	"context"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+	ctx := context.TODO()
+	config := hazelcast.Config{}
+	cc := &config.Cluster
+	cc.Network.SetAddresses("192.168.1.101:5701")
+	cc.Name = "hello-world"
+	creds := &cc.Security.Credentials
+	creds.Username = "member1"
+	creds.Password = "s3crEt"
+	client, err := hazelcast.StartNewClientWithConfig(ctx, config)
+	if err != nil {
+		panic(err)
+	}
+	client.Shutdown(ctx)
+}
+----
+. Run `go run main.go` in the terminal.
+
+Java::
++
+
+. Install the xref:getting-started:install-hazelcast.adoc#use-java[Java client library].
+. In your preferred Java IDE, create a new project to include a class containing the following code.
++
+[source,java]
+----
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+
+public class SecuredClient {
+  public static void main(String[] args) {
+
+ClientConfig clientConfig = new ClientConfig();
+        clientConfig.setClusterName("hello-world");
+        clientConfig.getSecurityConfig().setUsernamePasswordIdentityConfig("member1","s3crEt");
+        HazelcastClient.newHazelcastClient(clientConfig);
+
+  }
+}
+----
+. Run the `SecuredClient` class in the IDE.
+
+C Sharp::
++
+
+. Install the latest http://hazelcast.github.io/hazelcast-csharp-client/4.0.1/doc/obtaining.html[C Sharp client library]
+. In your preferred C# IDE, create a new project to include a class containing the following code.
++
+[source,cs]
+----
+var username = "member1";
+var password = "s3crEt";
+
+var options = new HazelcastOptionsBuilder();
+    .With(o => {
+        o.Authentication.ConfigureUsernamePasswordCredentials(username, password);
+    })
+    .Build();
+
+var client = await HazelcastClientFactory.StartNewClientAsync(options);
+----
+. Run this class in the IDE.
+
+C++::
++
+
+. Install the latest https://github.com/hazelcast/hazelcast-cpp-client/blob/v4.1.0/Reference_Manual.md#11-installing[C++ client library]
+. In your preferred C++ IDE, create a new project to include a class containing the following code.
++
+[source,cpp]
+----
+    hazelcast::client::client_config clientConfig;
+
+    clientConfig.set_credentials(
+            std::make_shared<hazelcast::client::security::username_password_credentials>("member1", "s3crEt"));
+
+    clientConfig.set_cluster_name("hello-world");
+    
+    auto hz = hazelcast::new_client(std::move(clientConfig)).get();
+----
+. Run this class in the IDE.
+
+Node.js::
++
+
+. Install the Node.js client library: `npm install hazelcast-client`
+. In your preferred Node.js IDE, create a new project to include the following script.
++
+[source,javascript]
+----
+const { Client } = require('hazelcast-client');
```

Comment:
We don't need any of this class.

We can just provide the authentication config to `Client.newHazelcastClient()`:

```
    security: {
        usernamePassword: {
            username: 'member1',
            password: 's3crEt'
        }
    }
```